manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -3620,6 +3620,11 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "watch", "list"]
+
+  #Used for MCS serviceexport management
+  - apiGroups: [""]
+    resources: ["serviceexports"]
+    verbs: ["create", "delete"]
```

Comment:
no `get` `list` needed?

Reply:
and watch, probably update and patch will be needed as well?

Reply:
This PR only needs create/delete on serviceexports. The eventual PR to consume them will likely need the ability to watch, but I want to add the minimal set of permissions necessary for this functionality.